adagrams/game.py

```diff
@@ -1,13 +1,121 @@
 from random import randint
 
+HAND_SIZE = 10
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+SCORE_CHART = {
+    ('A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T') : 1,
+    ('D', 'G') : 2,
+    ('B', 'C', 'M', 'P') : 3,
+    ('F', 'H', 'V', 'W', 'Y') : 4,
+    ('K') : 5,
+    ('J', 'X') : 8,
+    ('Q', 'Z') : 10
+}
+
+# Build a hand of 10 letters for the user
 def draw_letters():
-    pass
+    letters = []
+    total_pool = []
+    
+    for letter in LETTER_POOL:
+        for how_many in range(LETTER_POOL[letter]):
+            total_pool.append(letter)
+    
+    for i in range(HAND_SIZE):
+        r = randint(0, len(total_pool) - 1)
+        this_letter = total_pool.pop(r)
+        letters.append(this_letter)
+
+    return letters
 
+# Input can be made from user's hand
 def uses_available_letters(word, letter_bank):
-    pass
+    letter_bank_copy = []
+    word = word.upper()
+
+    for letter in letter_bank:
+        letter_bank_copy.append(letter)
```

Comment:
`letter_bank_copy` could be made in the same way with more concise code by using the `list` constructor.
```suggestion
    letter_bank_copy = list(letter_bank)
    word = word.upper()
```